Software/VNA_embedded/Application/Hardware.hpp

```diff
@@ -7,7 +7,7 @@
 #include "max2871.hpp"
 #include "Si5351C.hpp"
 
-#define USE_DEBUG_PINS
+#define USE_DEBUG_PINS	0
 
 #ifdef USE_DEBUG_PINS
```

Comment:
I believe this line should also be changed to "#if USE_DEBUG_PINS"

Reply:
@jankae you're correct, don't know why I didn't see it ;), it's fixed now.